app/graphql/mutations/users/mutation_manifest.rb

```diff
@@ -0,0 +1,12 @@
+module Mutations
+  module Users
+    module MutationManifest
+      extend ActiveSupport::Concern
+      included do
+        field :update_access_token, resolver: Mutations::Users::UpdateAccessTokenMutation
+        field :update_github_username, resolver: Mutations::Users::UpdateGithubUsernameMutation
+        field :update_github_org, resolver: Mutations::Users::UpdateGithubOrgMutation
```

Comment:
I don't want to lose the ability to update one at a time. It seems like a small overhead to have the extra POSTs, but saves a lot of headache especially since the form fields clear on submit.